pyfolio/utils.py

```diff
@@ -243,6 +243,8 @@ def extract_rets_pos_txn_from_zipline(backtest):
     """
 
     backtest.index = backtest.index.normalize()
+    if backtest.index.tzinfo is None:
+        backtest.index = backtest.index.tz_localize('UTC')
```

Comment:
In this case .tz_localize is fine because tzinfo has to be None. tz_localize works for tz-naive timestamps, and .tz_convert only works for tz-aware ones.